.github/workflows/ci.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         with:
           name: Clippy Check
           token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-features -- -D warnings
+          args: --all-features -- -D warnings -A clippy::upper_case_acronyms
```

Comment:
Although this seems to work for CI, I don't think it is the right way. I think we want a crate-level disabling, so it applies when you run `cargo clippy` outside of our CI workflow.

https://stackoverflow.com/questions/25877285/how-to-disable-unused-code-warnings-in-rust

Reply:
I agree. I think we're better off just creating a [clippy.toml](https://github.com/rust-lang/rust-clippy#configuration) at the project root and provide any global overrides in there. Disabling just for CI won't help with pre-commit hooks, for instance.

Reply:
unfortunately clippy.toml only applies to lints with configurable options, not individual lints https://www.reddit.com/r/rust/comments/eyjq12/clippy_configuration_options/fghuckg?utm_source=share&utm_medium=web2x&context=3

Hopefully Rick's PR will work for us and I'll remove this argument.